mvp-spec/operator-api.md

```diff
@@ -15,50 +15,87 @@ An extra cookie, with a very short lifetime, can be created to test the support
 
 | Cookie name       | Format                                 | Created by                     |
 |-------------------|----------------------------------------|--------------------------------|
-| `paf_identifier`  | [identifier.md](model/identifier.md)   | operator                       |
+| `paf_identifiers` | [identifiers.md](model/identifiers.md) | operator                       |
 | `paf_preferences` | [preferences.md](model/preferences.md) | contracting party, usually CMP |
-| `paf_test_3pc`    | <mark>TODO</mark>                      | operator                       |
+| `paf_test_3pc`    | [test-3pc.md](model/test-3pc.md)       | operator                       |
 
-### Example: identifier
+### Example of a `paf_identifiers` cookie
 
-<!--partial-begin { "files": [ "id.json" ], "block": "json" } -->
+<!--partial-begin { "files": [ "ids_cookie.txt" ], "block": "" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```
+[{"version":0,"type":"paf_browser_id","value":"7435313e-caee-4889-8ad7-0acd0114ae3c","source":{"domain":"operator.pafdemo.com","timestamp":1642504380,"signature":"MDhFhpZo3X39XD7BSwxnGvxX57h4qeG9M13WY9dKm/vHZP1qeP6iEVofdksbxD8p22wO5XYsHoLeixuLw8BfrA=="}}]
+```
+<!--partial-end-->
+
+Which is the "stringified" version of:
+
+<!--partial-begin { "files": [ "ids_cookie-pretty.json" ], "block": "json" } -->
 <!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
 ```json
-{
-  "version": 0,
-  "type": "prebid_id",
-  "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
-  "source": {
-    "domain": "operator.prebidsso.com",
-    "timestamp": 1642504380000,
-    "signature": "RYGHYsBUEwMgFgOJ9aUQl7ywl4xnqdmwWIgPbaIowbXbmZAFKLa7mcBJQuWh1wEskpu57SHn2mmCF6V5+cESgw=="
+[
+  {
+    "version": 0,
+    "type": "paf_browser_id",
+    "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+    "source": {
+      "domain": "operator.pafdemo.com",
```

Comment:
minor, could be FAR if the name already exists elsewhere: pafdemo.com is confusing considering we're also preparing a PAF demo. Suggestion : paf-operation-domain.io

Reply:
done